sdk/go/src/sawtooth_sdk/fake_processor.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	zmq "github.com/pebbe/zmq4"
+	"sawtooth_sdk/protobuf"
+)
+
+func main() {
+	context, _ := zmq.NewContext()
+	socket, _ := context.NewSocket(zmq.REQ)
+	defer context.Term()
+	defer socket.Close()
+
+	endpoint := "tcp://localhost:40000"
+	fmt.Printf("Connecting to %v...", endpoint)
+	socket.Connect(endpoint)
+	fmt.Println("done")
+
+	// Send registration request
```

Comment:
If we keep this file in this PR, let's update it to send a message the validator can understand.  Something like:

```	// Send registration request
	request := &protobuf.TpRegisterRequest{
		Family:     "test_family",
		Version:    "1.0",
		Encoding:   "protobuf",
		Namespaces: []string{"test_namespace"},
	}
	requestData, err := proto.Marshal(request)
	if err != nil {
		fmt.Printf("Failed to marshal: %v\n", err)
	}

	correlationId := "123"
	msg := &protobuf.Message{
		MessageType: protobuf.Message_TP_REGISTER_REQUEST,
		CorrelationId: correlationId,
		Content: requestData,
	}

	msgData, err := proto.Marshal(msg)
	if err != nil {
		fmt.Printf("Failed to marshal: %v\n", err)
	}
```